nut/rootfs/etc/cont-init.d/nut.sh

```diff
@@ -71,8 +80,8 @@ if bashio::config.equals 'mode' 'netserver' ;then
         fi
     done
 
-    if bashio::config.has_value "upsd_maxage"; then
-        maxage=$(bashio::config "upsd_maxage")
+    if bashio::config.has_value 'upsd_maxage'; then
+        maxage=$(bashio::config 'upsd_maxage')
```

Comment:
There are unrelated changes in this PR, can you please revert that?

Reply:
I have moved those changes to #473